pkg/apis/operations/v1alpha1/constants/types_constants.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
Let's keep our constants in `apis/core/v1beta1/constants`. Actually, the location is not quite clear and I plan to relocate it soon so that it doesn't make the impression that it's bound to the `core/v1beta1` package.